pypesto/objective/petab_import.py

```diff
@@ -197,7 +203,8 @@ def compile_model(self):
             observables=observables,
             constantParameters=constant_parameter_ids,
             sigmas=sigmas,
-            noise_distributions=noise_distrs
+            noise_distributions=noise_distrs,
+            *args, **kwargs
```

Comment:
one thing we would need to keep in mind: it may happen that there are other optional parameters which are not supposed to be forwarded to amici. but fine for the moment. whata is amici's behavior about arguments: will it raise if it gets an argument it does not understand?

Reply:
Yes, any additional arguments will have to be put there explicitly. In that call tree, I didn't see any function with a large number of arguments where a catch-all might be more appropriate than here. Can be adapted once that changes.

Yes, it will raise if there are non-conforming arguments.